src/palace/manager/data_layer/circulation.py

```diff
@@ -44,6 +46,14 @@ class CirculationData(BaseMutableData):
     last_checked: datetime.datetime | None = None
     should_track_playtime: bool = False
 
+    # The licensing model for the pool (METERED, UNLIMITED, or AGGREGATED).
```

Comment:
Minor (feel free to ignore): Not sure we should enumerate the types here, since they might change over time and would need to be kept in sync with the enum.